webapp/tests/elster_client/mock_erica.py

```diff
@@ -27,10 +27,11 @@
 
 
 class MockResponse:
-    def __init__(self, json_data, status_code):
+    def __init__(self, json_data, status_code, location=None):
```

Comment:
this can be removed or?

Reply:
Nop, it is needed when posting a job with the MockErica so that the location is set in the headers